sensing/autoware_pointcloud_preprocessor/include/autoware/pointcloud_preprocessor/concatenate_data/concatenate_and_time_sync_node.hpp

```diff
@@ -49,19 +49,22 @@
  *
  */
 
```

Comment:
Same comment as in the other files: as far as I understand, this is a complete rewrite, so removing this license is ok